### DISCUSSION TITLE
How to open the Visual Studio Code Command Palette in a GitHub codespace

### DISCUSSION THREAD

**Discussion by EvanAad** [community/community#65002] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I'm trying to follow [this guide](https://docs.github.com/en/codespaces/setting-up-your-project-for-codespaces/adding-a-dev-container-configuration/setting-up-your-java-project-for-codespaces) for creating a Java codespace. Step 1 reads:

> Access the Visual Studio Code Command Palette (Shift+Command+P / Ctrl+Shift+P), then start typing "add dev".

However, Shift+Command+P makes my FireFox browser open a new incognito browser window, whereas Ctrl+Command+P makes my Windows 10 Pro operating system open a side panel for configuring a second screen.

So how can I access the VS Code Command Pallette?

**Comment by arafajs** [comment, marked as answer]
The keyboard shortcut Shift+Command+P is a reserved shortcut in Firefox, so it is not possible to use it to open the VS Code Command Palette.

The following are the other ways to open the VS Code Command Palette:

- From the menu bar, select View > Command Palette.
- Press Ctrl+Shift+P on Windows or Linux.
- Press Command+Shift+P on macOS.
- You can also use the keyboard shortcut Ctrl+K followed by P to open the Command Palette.

Once the Command Palette is open, you can start typing the name of the command you want to run. For example, to run the add dev command, you would type add dev and then press Enter.

Here are some other commands that you can run from the Command Palette:

- Open file: Opens a file in the editor.
- Save file: Saves the current file.
- Run code: Runs the current file in the debugger.
- Install extension: Installs an extension from the Visual Studio Code Marketplace.
- Search: Searches for files, symbols, and commands.

You can find a list of all the commands available in the Command Palette by pressing Ctrl+Shift+P and typing list commands.

**Reply by EvanAad** [reply]
Thanks. Of all the methods you described, the only one that works for me is the one that uses the View menu.
